ogm/faq.adoc

```diff
@@ -183,6 +183,27 @@ Dependencies: org.hibernate:ogm services, org.hibernate.ogm.mongodb services
 Alternatively, you can configure this via the descriptor _jboss-deployment-structure.xml_.
 See the https://docs.jboss.org/hibernate/ogm/4.1/reference/en-US/html_single/#ogm-configuration-jbossmodule[reference guide] to learn more.
 
+=== Why do I get a `java.lang.NoClassDefFoundError: org/hibernate/search/batchindexing/spi/MassIndexerFactory`?
+
+This means that you need Hibernate Search on the class path.
+You can add it via maven using the following coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-search-orm</artifactId>
+</dependency>
+----
+
+You can include the right version using the BOM as describe in the
+https://docs.jboss.org/hibernate/stable/ogm/reference/en-US/html/ch02.html[documentation getting started].
```

Comment:
"documentation getting started" looks weird. "getting started chapter of the documentation" maybe?